compiler/src/Language/Bond/Codegen/Cpp/Enum_h.hs

```diff
@@ -20,7 +20,7 @@ enum_h :: MappingContext -> String -> [Import] -> [Declaration] -> (String, Text
 enum_h cpp _file _imports declarations = ("_enum.h", [lt|
 #pragma once
 
-#include <stdint.h>
+#include <cstdint>
```

Comment:
While `<cstdint>` may be more C++ish, we use the unnamespaced `<stdint.h>` type names everywhere else and consuming code may be doing so as well. This doesn't seem like we get enough benefit to make that breaking change.

Reply:
I looked through the current versions of `<cstdint>` in MSVC, gcc, and clang and all three would be fine with this code.

However, you are write, the consuming code might not be on the most recent toolchain. I will revert.